web/src/composebox_typeahead.ts

```diff
@@ -1231,10 +1241,14 @@ export function content_typeahead_selected(
                 beginning = beginning.slice(0, syntax_start_index) + "#**" + stream_name + "** ";
                 break;
             }
-            beginning =
-                beginning.slice(0, syntax_start_index) +
-                topic_link_util.get_stream_topic_link_syntax(stream_name, item.topic) +
-                " ";
+            const stream_topic_link_syntax =
+                item.topic_display_name === util.get_final_topic_display_name("")
```

Comment:
In think ideally, we'd have the original `topic` on the object, and be comparing `topic === ""`, but I'm not sure it's very important.

Reply:
It probably doesn't matter, this can be as-is.